src/payment/strategies/afterpay/afterpay-payment-strategy.ts

```diff
@@ -98,7 +101,13 @@ export default class AfterpayPaymentStrategy implements PaymentStrategy {
 
         await this._store.dispatch(this._orderActionCreator.submitOrder({}, options));
 
-        return this._store.dispatch(this._paymentActionCreator.submitPayment(paymentPayload));
+        return this._store.dispatch(this._paymentActionCreator.submitPayment(paymentPayload)).catch(error =>
```

Comment:
Can you use `async/await`? Also, does this error handling flow apply to all kinds of error?

Reply:
@davidchin thanks for your feedback 🙇 
I've applied the suggested changes, and as for the question above ... yes, this error handling applies to all errors that might come from submitting the payment